### DISCUSSION TITLE
[FE] interfaceかtypeか

### DISCUSSION THREAD

**Discussion by Rinasham** [Hackathon5-TeamD/evolution-quest#45] [Q&A]
interfaceとtypeのどちらを使えばいいのかよく分かんないので、こっちの方が使いやすいぜ！みたいなのあったら教えてください（*’∀’人）

**Comment by Terralien-jp** [comment]
よくわからんので
https://zenn.dev/luvmini511/articles/6c6f69481c2d17

**Comment by kobakichi** [comment]
React芸人のトラハック氏によると、ケースバイケースみたいですが、若干typeの方が良さそうと言う感じでしょうか？

https://youtu.be/J2vox52T4W8

https://youtu.be/2DoYdw-rvL0

私はtypeしか使った事ありませんが。。、

**Comment by haru-24** [comment]
自分はinterfaceばかり使ってますね、、、笑
記事とかみてるとType推しが多いようですね🧐

**Comment by Rinasham** [comment]
個人の趣味的なチョイスでよしって感じでしょうか？w
小林さんの好きな方に合わせます(﹡ˆᴗˆ﹡)

**Reply by kobakichi** [reply]
特に理由はありませんが、使った事があるという事でtypeで今回は行きましょう！！！
